5.0 Draft/IDD/IDDs Orchestration System/eu.arrowhead.orchestration-pull-http-json.yml

```diff
@@ -0,0 +1,303 @@
+        and `unix`. A protocol only counts as a transport if it both (A) provides a way of
+        addressing and, by extension, sending messages to service providers and consumers, as well
+        as (B) does not build upon another protocol also providing this capability. I other words,
+        TLS and DTLS are _not_ transports, because they build upon the TCP and UDP protocols, which
+        satisfy condition A.
+
+        What the _location_ part consists of depends on what transport is identified. If the
+        transport is `tcp4` or `udp4`, the location is an IPv4 address expressed as four decimal
+        numbers separated by dots, a colon and a port number, such as in `192.168.3.22:64075`. If
+        the transport is `tcp6` or `udp6`, the location is an IPv6 address, rendered as described in
+        [RFC 5952](https://www.rfc-editor.org/rfc/rfc5952), within square brackets, followed by a
+        colon and a port number. If the transport is `unix`, the location is an absolute filesystem
+        path to a Unix socket file.
+      type: string
+      pattern: ^(?<transport>[^:]+):\w*(?<location>.*)$
+      example: tcp4:192.168.0.7:45326
+
+    Auth-tokens:
+      description: |
+        An (k,v) object containing authorization tokens for each interface contained in the response
+      type: object
+      example: {"interfaceName1":"token1"}
+
+    interface:
+      description: A string that describes the interface in Protocol-SecurityType-MimeType format. SecurityType can be either SECURE or 
+        INSECURE. 
+      type: string
+      pattern: ^([a-zA-Z0-9_-]+)-(SECURE|INSECURE)-([a-zA-Z0-9_-]+)$
+      example: HTTP-SECURE-JSON
+    
+    Metadata:    
+      description: |
+        Additional details of relevance when consuming a service. What metadata is made available
+        depends on the type of the service holding the metadata. Possible examples of metadata are
+        HTTP base paths, message size restrictions and caching directives.
+      type: object
+      example: {"basePath":"v2"}
+
+    PreferredProvider:
+      type: object
+      description: information of a preferred provider to consume services from.
+      properties:
+        provider-cloud:
+          $ref: '#/components/schemas/ProviderCloud'
+        provider:
+          $ref: '#/components/schemas/ProviderSystem'
+
+    ProviderSystem:
+      type: object
+      description: information describing a provider system within a local cloud. 
+      properties:
+        system-name:
+          type: string
+        address:
+          $ref: '#/components/schemas/Address'
+    
+    ProviderCloud:
+      type: object
+      description: an object describing the cloud in which a provider is located
+      properties:
+        operator:
+          type: string
+          description: Name of the operator company
+        name: 
+          type: string
+          description: name of the cloud 
+
+    RequesterSystem:
```

Comment:
I don't see why you would need to provide most of this information. The unique ID of the requesting system should be enough, right?

Reply:
Would you say its incorrect to have these "input forms" where not all data is required? Should these IDs be the same as in the system registry? or should they be unique identifiers built from their information? Should consuming systems first query a system registry for their ID and then query the orchestrator?